dask/sizeof.py

```diff
@@ -84,6 +84,13 @@ def sizeof_pandas_index(i):
             p += object_size(i)
         return int(p) + 1000
 
+    @sizeof.register(pd.MultiIndex)
+    def sizeof_pandas_multiindex(i):
+        p = int(sum(object_size(l) for l in i.levels))
+        for c in i.codes if hasattr(i, "codes") else i.labels:
```

Comment:
Can you make this an `if PANDAS_GT_0240` rather than a hasattr? Makes it easier to cleanup in the future. Thats in `dask.dataframe.utils`.

Reply:
I'm curious about this request.  It makes more sense to me to check the object itself, rather than depend on an understanding of what was added in which version.

Ah!  Perhaps you're looking forward to the time when we can remove the check because we've dropped support for 0.24, yes?

Reply:
> Ah! Perhaps you're looking forward to the time when we can remove the check because we've dropped support for 0.24, yes?

Exactly. I've started making this request in a few places, without checking with other maintainers. I'll make a discussion issue when I get a chance.

Reply:
@TomAugspurger I like the idea but in this case importing `PANDAS_GT_0240` gives a circular import error. any suggestion on how to resolve this?